sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -600,20 +593,20 @@ async def update_repository_properties(
 
     @overload
     async def update_manifest_properties(
-        self, repository: str, tag_or_digest: str, properties: "ArtifactManifestProperties", **kwargs: Any
-    ) -> "ArtifactManifestProperties":
+        self, repository: str, tag_or_digest: str, properties: ArtifactManifestProperties, **kwargs
+    ) -> ArtifactManifestProperties:
         ...
 
     @overload
     async def update_manifest_properties(
-        self, repository: str, tag_or_digest: str, **kwargs: Any
-    ) -> "ArtifactManifestProperties":
+        self, repository: str, tag_or_digest: str, **kwargs
```

Comment:
Same comment as the sync client - we should update these `update_` overloads to include the expand keyword-only args.